### ISSUE TITLE
Team update - Nov 24, 2021

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#62]
This is a  @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with XXX...
- Thanks for Jo's work on the XXX repo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>
- I had a meeting with ABC

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...
- Could somebody take a look at ...

**My availability for next week**
- I'll be off on XXX day...
- I've got several meetings this week...
```

</details>

#### :mag: **Needs Triage**:

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

* [Investigate accessibility statements](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/52)

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- Isabela and Tony for their impromptu show and tell at our team check-in. Loved getting teasers about the Paris commune, design work in union organizing, and such.
- Also, thanks to Isabela and Tony for following up with me after the JupyterLab calls on Wednesday

**Updates from last week :heavy_check_mark:**
- Since our check-in, I switched gears to work on Qhub stuff, so I don't have a lot for this update. I'll be spending the remaining two work days this week on https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/4 and https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/51
- I will have a meeting with Frédéric Collonval on Friday to discuss testing

**Challenges I faced and things I'd like assistance with 🙏**
- The amount of stuff all at the same time in Jupyter land is hard to wrap one's mind around. Isabela and Tony are doing a great job helping to orient me to this vast and complex landscape. I will need to continue to rely on them for this for a while.

**My availability for next week**
- I'll be available all next week, but switching time zones! I'll be in Eastern Standard Time (UTC-5), visiting with friends for two weeks in Durham and working out of the American Underground coworking space Monday-Friday, roughly 9 to 5 Eastern!
